user_groups/user_groups.md

```diff
@@ -298,9 +299,62 @@ Response
 ```
 ---
 
-## Add or update user group permission for a given resource
+## Add user group permission for a given resource
 
-Add or update resource permission of a user group.
+Add resource permission of a user group.
+
+```apacheconfig
+POST /v1/usergroups/{userGroupId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to request. |
+| resourceId | `string` | The resource id to add. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+```json
+{
+  "resourceId": "p-2343",
+  "role": "reader"
+}
+```
```

Comment:
@WMKhant similar to what we did for user APIs, we should put a table with the Name, Type and Description for each fields in the JSON and this should be an example. Please see users.md for more details.

Reply:
added the table

Reply:
Role type is wrong, it should be enum.

Reply:
fixed